### PR TITLE
Fixing link to contrib guidelines

### PR DESCRIPTION
**Pull request by spekulatius**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 This list is mainly about JavaScript - the language. Not about APIs, tooling, frameworks or other aspects of todays JavaScript ecosystem.
 
-*Please read the [contribution guidelines](.github/contributing.md) before contributing.*
+*Please read the [contribution guidelines](https://github.com/micromata/awesome-javascript-learning/blob/master/contributing.md) before contributing.*
```
Comment (mischah) [inline readme.md:7]:
```suggestion
*Please read the [contribution guidelines](contributing.md) before contributing.*
```

**Review by mischah — CHANGES_REQUESTED** [review]
Hej, thanks for the PR addressing #43 :octocat: 

Please use a relative link to fix this issue.

**Comment by spekulatius** [comment]
Hey @mischah, it's updated :+1:

**Review by mischah — APPROVED** [review]
Yeah 💯

**Pull request merged**